lib/egauge/power_helpers.rb

```diff
@@ -10,6 +10,21 @@ def monthly_kwh(number_of_months = 12)
       query(:C => nil, :T => months(number_of_months))
     end
 
+    def past_24_hrs_kwh(csv=false)
+      end_time = Time.now.beginning_of_hour
+      ar = []
```

Comment:
I'd expand `ar`, not sure what it stands for (but I bet once you tell me it'll be obvious).